### PR TITLE
completed css-brochure-form

### PR DESCRIPTION
**Pull request by josephswpark**
Screen Shot 2022-10-12 at 3 47 30 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Joseph, this form is WAY too wide as compared to the example. Please revise and submit a new screenshot with your revision.

**Comment by josephswpark** [comment]
Screen Shot 2022-10-12 at 3 53 06 PM

**Comment by josephhparkk** [comment]
Screen Shot 2022-10-12 at 3 02 02 PM

**Comment by josephswpark** [comment]
Screen Shot 2022-10-12 at 4 04 47 PM

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <input type="text" name="name" value="Sally Smith">
+
+    <label for="location">WHERE ARE YOU HIRING?</label>
+    <input type="text" name="location" value="City, Country">
+
+    <label for="hiring">HOW MANY ENGINEERS ARE YOU HIRIGN IN THE NEXT 6 MONTHS?
+    </label>
+    <select name="engineers" id="hiring">
+      <option value="1-2">1-2 Engineers</option>
+    </select>
```
Comment (codingcodymiller) [inline css-brochure-form/index.html]: These labels are not properly configured to pair with their form control. The form control elements need an `id` attribute with the same value as the `for` attribute on the label they are paired with.

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+body {
+  background-color: white;
+}
+
+h3 {
+  text-align: center;
+  font-family: "Brandon Grotesque";
+  font-style: normal;
+  font-weight: 500;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: These are only necessary on the `@font-face` rules I made, you don't need them to use the font family, just `font-family`

File: css-brochure-form/styles.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+label {
+  padding-left: 7px;
+  font-size: 60%;
+  text-align: center;
+  font-family: "Brandon Grotesque";
+  font-style: normal;
+  font-weight: bold;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: These are only necessary on the `@font-face` rules I made, you don't need them to use the font family, just `font-family`

File: css-brochure-form/styles.css [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+input {
+  width: 100%;
+  padding: 6px;
+  font-family: AtlasGrotesk;
+  font-style: normal;
+  font-weight: 300;
+  src: url("AtlasGrotesk-Light-Web.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: These are only necessary on the `@font-face` rules I made, you don't need them to use the font family, just `font-family`

File: css-brochure-form/styles.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  margin-top: 2.5px;
+  margin-bottom: 7.5px;
+}
+
+select {
+  width: 100%;
+  font-family: "Brandon Grotesque";
+  font-style: normal;
+  font-weight: normal;
+  src: url("BrandonGrotesque-Regular.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: These are only necessary on the `@font-face` rules I made, you don't need them to use the font family, just `font-family`

File: css-brochure-form/styles.css [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+  font-size: 65%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: You should use `px` or `rem`

File: css-brochure-form/styles.css [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+  color: white;
+  border: white;
+  background-color: rgb(236 97 91);
+  border-radius: 15px;
+  padding: 5px 12px;
+  font-family: "Brandon Grotesque";
+  font-style: normal;
+  font-weight: 500;
+  src: url("BrandonGrotesque-Regular.woff2") format("woff2");
+  font-size: 65%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: You should use `px` or `rem`

File: css-brochure-form/styles.css [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+input {
+  width: 100%;
+  padding: 6px;
+  font-family: AtlasGrotesk;
+  font-style: normal;
+  font-weight: 300;
+  src: url("AtlasGrotesk-Light-Web.woff2") format("woff2");
+  color: rgb(175 175 175);
+  font-size: 65%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: You should use `px` or `rem`

File: css-brochure-form/styles.css [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+form {
+  width: 20%;
+  margin: auto;
+}
+
+label {
+  padding-left: 7px;
+  font-size: 60%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: You should use `px` or `rem`

File: css-brochure-form/styles.css [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+button {
+  color: white;
+  border: white;
+  background-color: rgb(236 97 91);
+  border-radius: 15px;
+  padding: 5px 12px;
+  font-family: "Brandon Grotesque";
+  font-style: normal;
+  font-weight: 500;
+  src: url("BrandonGrotesque-Regular.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: These are only necessary on the `@font-face` rules I made, you don't need them to use the font family, just `font-family`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by josephswpark** [comment]
Screen Shot 2022-10-12 at 4 27 56 PM

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**